supla-common/proto.h

```diff
@@ -412,6 +413,7 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 #define SUPLA_BIT_FUNC_RAINSENSOR 0x00002000                // ver. >= 12
 #define SUPLA_BIT_FUNC_WEIGHTSENSOR 0x00004000              // ver. >= 12
 #define SUPLA_BIT_FUNC_CONTROLLINGTHEROOFWINDOW 0x00008000  // ver. >= 13
+#define SUPLA_BIT_FUNC_CONTROLLINGTHEFACADEBLIND 0x00010000  // ver. >= 17
```

Comment:
I prefer the format 0x0010000

Reply:
Current format is aligned with other BIT_FUNC entries above. Are you sure? :)
If yes, would you like me to align all above BIT_FUNC as well to have one digit less?